casdoc_test.go

```diff
@@ -15,78 +15,53 @@
 package main
 
 import (
-	"fmt"
 	"path"
 	"strings"
 	"testing"
 
 	"github.com/casbin/casdoc/config"
 	. "github.com/casbin/casdoc/logger"
 	"github.com/casbin/casdoc/utils"
-	log "github.com/sirupsen/logrus"
 )
 
 func TestPolishDocs(t *testing.T) {
 	Q := utils.WorkQueue{}
 	err := Q.GetFileList(path.Join(config.RepoPath, "/docs/"))
 	if err != nil {
-		Logger.Errorf("Failed to get file list")
-		panic(err)
+		t.Fatalf("Failed to get file list, err: %v", err)
```

Comment:
In unit testing, I believe using the `t.Fatalf` to terminate the test is more appropriate for the scenario.